server/enterprise/message_export/actiance_export/actiance_export.go

```diff
@@ -235,13 +302,100 @@ func channelHasActivity(cmhs []*model.ChannelMemberHistoryResult, startTime int6
 	return false
 }
 
-func postToExportEntry(post *model.MessageExport, createTime *int64, message string) *PostExport {
+func createdPostToExportEntry(post *model.MessageExport) PostExport {
```

Comment:
Here and in a few methods below I assume we are guaranteeing post can't be `nil`.